app/models/name_format_validator.rb

```diff
@@ -0,0 +1,11 @@
+class NameFormatValidator < ActiveModel::EachValidator
+  NAME_REGEX_FILTER = /\A[^"=$%#&*+\/\\()@?!<>0-9]*\z/
```

Comment:
it's probably not in the scope of this PR, but this regex seems too lenient.
It's allowing the characters: []^_`|{}~

Reply:
ah good spot! this is taken from the current personal details logic. i'll raise a new PR so it can be product reviewed separately 👍